### PR TITLE
Add exception for debian 6 based things

### PR DESCRIPTION
**Pull request by scotthain**
@chef-cookbooks/engineering-services

### CODE REVIEW

**Comment by jtimberman** [comment]
:lollipop: :+1:

File: libraries/debian_handler.rb [hunk 1]
```diff
@@ -43,7 +43,8 @@ def configure_package(action_name)
       # we're all sad about this, but ubuntu 10.04 will fail with a message
       # about using this option if we don't do it here.
```
Comment (schisamo) [inline libraries/debian_handler.rb]: We should update this comment.

**Comment by scotthain** [comment]
@schisamo Thanks, updated the comment

**Comment by yzl** [comment]
:+1:

**Pull request merged**